website/docs/reference/integrations/slack-app.md

```diff
@@ -0,0 +1,64 @@
+---
+
+:::info Availability
+
+The Slack App integration was introduced in **Unleash 5.5**.
+
+:::
+
+The Slack App integration posts messages to the selected channels in your Slack workspace.
+
+## Installation {#installation}
+
+To install the Slack App integration, follow these steps:
+
+1. Navigate to the *integrations* page in the Unleash admin UI (available at the URL `/integrations`) and click "configure" on the Slack App integration.
+2. On the integration configuration form, click the "install & connect" button.
+3. A new tab will open, asking you to select the Slack workspace where you'd like to install the app.
+4. After successful installation of the Unleash Slack App in your chosen Slack workspace, you'll be automatically redirected to a page displaying a newly generated access token.
+5. Copy this access token and paste it into the `Access token` field within the integration settings.
+
+By default, the Unleash Slack App is granted access to public channels. If you want the app to post messages to private channels, you'll need to manually invite it to each of those channels.
+
+## Configuration {#configuration}
+
+The configuration settings allow you to choose the events you're interested in and whether you want to filter them by projects and environments. You can configure a comma-separated list of channels to post the configured events to. These channels are always notified, regardless of the event type or the presence of tags.
+
+#### Events {#events}
+
+You can choose to trigger updates for the following events:
+
+- feature-created
+- feature-metadata-updated
+- feature-project-change
+- feature-archived
+- feature-revived
+- feature-strategy-update
+- feature-strategy-add
+- feature-strategy-remove
+- feature-stale-on
+- feature-stale-off
+- feature-environment-enabled
+- feature-environment-disabled
+- feature-environment-variants-updated
+- feature-potentially-stale-on
+
+#### Parameters {#parameters}
+
+The Unleash Slack App integration takes the following parameters.
+
+- **Access token** - This is the only required property. After successful installation of the Unleash Slack App in your chosen Slack workspace, you'll be automatically redirected to a page displaying a newly generated access token. You should copy this access token and paste it into this field.
+- **Channels** - A comma-separated list of channels to post the configured events to. These channels are always notified, regardless of the event type or the presence of a slack tag.
+
+## Tags {#tags}
+
+Besides the configured channels, you have the flexibility to choose to notify other channels by tagging your feature toggles with Slack-specific tags. For instance, if you want the Unleash Slack App to send notifications to the `#general` channel, simply configure your feature toggle with the "slack:general" tag. This will ensure that any configured events related to that feature toggle will notify the tagged channel in addition to any configured channels.
```

Comment:
Another minor thing: I think we're moving to using "feature flags" instead of "feature toggles", so I think that in new documents, we should try and stick to that. But maybe @dgorton has a different opinion?

Also, more importantly, I think we should explain tags a little better. "slack:general" doesn't tell me explicitly that it's a tag of type "slack" with the value "general". Also, does it accept both "#general" and "general"? I know that I forget which one is right every time.

Also, "any configured channels" means configured for the entire instance, right? Maybe that could be a little clearer too.

Maybe something like:

```suggestion
Besides the configured channels, you can choose to notify other channels by tagging your feature flags with Slack-specific tags. For instance, if you want the Unleash Slack App to send notifications to the `#general` channel, add a slack-type tag with the value "general" (or "#general"; both will work) to your flag. This will ensure that any configured events related to that feature flag will notify the tagged channel in addition to any channels configured on the integration-level.
```

Reply:
Oh, I like this! I went to double-check whether `#channels` would work on both the integration configuration and tags, and good news is that they do!

However I caught a bug where it would not be possible to delete such a tag (e.g. "slack:#general") so as a slight side-effect of this PR I fixed it [here](https://github.com/Unleash/unleash/pull/4801/commits/328defd7fe305e700fb7a157fa5996f31a622087).